client/app/hosting/database/dump/hosting-database-dump.controller.js

```diff
@@ -123,7 +106,7 @@ angular.module("App").controller(
 
             if (dump && dump.id) {
```

Comment:
`_(dump).isObject() && _(dump).isFinite(dump.id)`

Reply:
first, ```if (_(dump.get(id))``` would be a lot simpler
second, this is not part of this pull request
third, lowdash is making our code bloated